lib/firebase-admin.ts

```diff
@@ -0,0 +1,10 @@
+import * as admin from "firebase-admin";
```

Comment:
Just curious why separate this out to its own file?

Reply:
Thought it might be smarter to have a separate file for firebase admin logic so we're explicit about it (and careful) especially because admin functionality has more access to tables

Plus the firebase-admin is _slightly_ different in how it makes calls to firestore